test/unit/gie_self_tests.cpp

```diff
@@ -442,6 +442,12 @@ TEST(gie, info_functions) {
     /* we can't expect perfect numerical accuracy so testing with a tolerance */
     ASSERT_NEAR(-2.0, proj_dmstor(&buf[0], NULL), 1e-7);
 
+    /* test UTF-8 degree sign on DMS input */
+    ASSERT_NEAR(0.34512432, proj_dmstor("19°46'27\"E", NULL), 1e-7);
+
+    /* test ISO 8859-1, cp1252, et al. degree sign on DMS input */
+    ASSERT_NEAR(0.34512432, proj_dmstor("19\26046'27\"E", NULL), 1e-7);
```

Comment:
MSVC complains about warning C4125: decimal digit terminates octal escape sequence
Perhaps the following would do ?
```suggestion
    ASSERT_NEAR(0.34512432, proj_dmstor("19" "\260" "46'27\"E", NULL), 1e-7);
```